lib/active_merchant/billing/gateways/moneris.rb

```diff
@@ -182,16 +185,16 @@ def expdate(creditcard)
 
       def add_payment_source(post, payment_method, options)
         if payment_method.is_a?(String)
-          post[:data_key]   = payment_method
-          post[:cust_id]    = options[:customer]
+          post[:data_key] = payment_method
+          post[:cust_id] = options[:customer]
         else
           if payment_method.respond_to?(:track_data) && payment_method.track_data.present?
-            post[:pos_code]   = '00'
-            post[:track2]     = payment_method.track_data
+            post[:pos_code] = '00'
```

Comment:
It's probably not worth going back and undoing, but the whitespace alignment changes here (and in the rest of this file) make the diff appear much larger than it actually needed to be, and make it a bit harder to find the actual logic changes. (They also shouldn't be necessary; it's fine if you found it annoying, but RuboCop should not have complained, since it allows extra whitespace specifically for this kind of alignment.) It'd be to keep logic and whitespace changes separate when they easily can be in the future.

Reply:
👍 
Rubocop had lots to say about these files and I thought reformatting the file would help. It did some, but there were lots of adjustments like these. I agree that that it is easier to keep logic changes separate and intended to take these out this morning. I hadn't requested any reviewers yet for just that reason. Since you have already approved, and waded through the extra long diff, I will just leave them in.

Reply:
RuboCop shouldn't have complained about any of these, though. I actually do want to figure out why that's happening, since it implies we might've configured RuboCop wrong.

Reply:
Rubocop is fine. Its complaints were about my additions (lots of options massaging) and reformatting addressed most of the issues I introduced. It also deleted all the extra spaces, which as I said, I intended to undo this morning.

Reply:
Oooh, got it! Sorry, I've clearly got a reading comprehension issue this morning. All good! 👍

Reply:
I do find it weird, I indeed don't see you having requested review, but I got pinged by GitHub on this this morning, and thought I saw @dtykocki and me tagged. I wonder if GitHub somehow sent emails to _recommended_ reviewers? Or something else?